tf-modules/ingest/outputs.tf

```diff
@@ -1,21 +1,31 @@
-output "discover_granules_task_lambda_function_arn" {
-  value = aws_lambda_function.discover_granules_task.arn
+output "discover_granules_task" {
+  value = {
+    task_arn : aws_lambda_function.discover_granules_task.arn
```

Comment:
So `=` and `:` both work within a terraform object? `workflow_config` in the file above uses `=` but this uses `:`

Reply:
Apparently, but I'll fix that.